developing-packages.md

```diff
@@ -173,3 +176,49 @@ packages pub publish --dry-run`. Finally, run the actual publish command:
 
 For details on publishing, see the [Pub publishing docs](https://www.dartlang.org/tools/pub/publishing).
 
+## Handling package interdependencies {#dependencies}
+
+If you are developing a package `hello` that depends on the Dart API exposed by another package,
+you need to add that package to the `dependencies` section of your `pubspec.yaml` file.
+The code below makes the Dart API of the `url_launcher` plugin available to `hello`:
+
+In `hello/pubspec.yaml`:
+```yaml
+:
```

Comment:
This looks weird on the generated page. Maybe use '...' instead, if that's what you're going for? (Same for the Android/iOS snippets below.)